File: beispiele/zahlen-raten/zahlen-raten.php
<?php
    session_start();
    if (!isset($_SESSION['zufallszahl'])) {
        $_SESSION['zufallszahl'] = rand(0, 101);
    } 

    $zahl = 23;
        $zufallszahl = $_SESSION['zufallszahl'];
    
    if ($zahl < $zufallszahl) {
        echo "Höher!";
    } elseif ($zahl > $zufallszahl) {
        echo "Niedriger!";
    } else {
        echo "Glückwunsch, du hast die richtige Zahl geraten!";
        unset($_SESSION['zufallszahl']);
        echo "<a href='zahlen-raten.php'>Neues Spiel starten</a>";
    }
?>

File: beispiele/zahlen-raten/zahlen-raten.py
import random

zufallszahl = random.randint(1, 100)
eingabe = 12

while eingabe != zufallszahl:
    if eingabe < zufallszahl:
        print("Höher!")
    elif eingabe > zufallszahl:
        print("Niedriger!")
    eingabe = int(input("Bitte gib eine neue Zahl ein: "))
print("Glückwunsch, du hast die richtige Zahl geraten!")